x-pack/plugins/cases/public/components/custom_fields/toggle/get_eui_table_column.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import { EuiSwitch } from '@elastic/eui';
+
+import type { CaseUI } from '../../../containers/types';
+import { CustomFieldTypes } from '../../../../common/types/domain';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getEuiTableColumn = ({
+  key,
+  label,
+}: {
+  key: string;
+  label: string;
+}): EuiTableComputedColumnType<CaseUI> => ({
+  name: label,
+  render: (theCase: CaseUI) => {
+    const index = theCase.customFields.findIndex((element) => element.key === key);
+
+    if (index === -1) {
+      return (
+        <EuiSwitch
```

Comment:
What is the latest status from @mdefazio about how we show this?

Reply:
I followed the comment [here](https://github.com/elastic/kibana/issues/167617#issuecomment-1781081636) and let icons. I like how it looks.

Reply:
Nice!